source/CardGames.py

```diff
@@ -153,24 +153,33 @@ def resetDeck(self):
     self.deck.reset()
     self.deck.shuffle()
 
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
+def character_selection():
+  Player1 = input("Player 1, please enter your name: ")
+  Player2 = input("Player 2, please enter your name: ")
+  Player1Role = ""
+  Player2Role = ""
+  rand = random.randint(0,1)
+  if rand == 0:
+    while Player1Role != "Fugitive" and Player1Role != "fugitive" and Player1Role != "Marshall" and Player1Role != "marshall":
```

Comment:
Great work brother! However, to reduce the amount of **and** statements, try using the lower() method to sanitize your input and make it lowercase. This ensures that all strings will be lowercase.

Reply:
Sounds good! I will be sure to use the lower method. Thanks for letting me know!